src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -82,8 +84,11 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
     window: any = window;
 
     componentDidMount() {
+        const { onLoad, versionUpsellExperience } = this.props;
         this.initialize();
-        this.fetchData();
+        this.fetchData().then(() => {
+            onLoad('preview', 'versions', { versionUpsellExperience });
```

Comment:
How about something like, `onLoad({ component: 'preview', feature: 'versions', versionUpsellExperience })`

Reply:
right now onLoad takes two strings and an object. Do you think we should have it only take an object, and put the component and feature strings in the data object?